test/test_jit.py

```diff
@@ -371,19 +371,23 @@ def assertExpectedONNXGraph(self, trace, *args, **kwargs):
         torch.onnx._optimize_trace(trace, operator_export_type=OperatorExportTypes.ONNX)
         self.assertExpectedGraph(trace, *args, **kwargs)
 
-    def assertExpectedGraph(self, trace, *args, **kwargs):
-        if isinstance(trace, torch._C.Graph):
-            graph = trace
-        else:
-            graph = trace.graph()
-
+    def sanitizeGraph(self, graph):
         torch._C._jit_pass_lint(graph)
         torch._C._jit_pass_dce(graph)
         torch._C._jit_pass_lint(graph)
         graph = torch._C._jit_pass_canonicalize(graph)
         torch._C._jit_pass_lint(graph)
+
+    def assertExpectedGraph(self, trace, *args, **kwargs):
```

Comment:
I think we can just delete this assertExpectedGraph since we are not using it?

Reply:
The last remaining usages of it are removed in a different PR, but not yet removed on master so leaving it in for now...